packages/types/src/localization.ts

```diff
@@ -610,6 +614,12 @@ type _LocalizationResource = {
       actionLabel__remove: LocalizationValue;
       formTitle: LocalizationValue;
       formSubtitle: LocalizationValue;
+      radioLabel__manualInvitation: LocalizationValue;
```

Comment:
Do we want these to be coupled with the radio element? I think these should be more generic

Reply:
How about `optionManualInvitation__label` ?

Reply:
@panteliselef `manualInvitationOption__label` probably? 

Also `manualInvitation__optionLabel` because we might have `manualInvitation__optionSubtext`, `manualInvitation__title`

Reply:
I went with @raptisj suggestion 🙏